html/frontend.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+<html lang="">
```

Comment:
I haven't observe this problem yet, but I think it does not harm to add it. I will do it.

Reply:
I went through the code one again and I assume we are good even without it.

Previously, we had
```html
<title>{{ $title }}</title>
@yield('head-js')
@yield('head-css')
<link rel="shortcut icon" href="favicon.ico">
```
(see [`resources/views/includes/head.blade.php`](https://github.com/LycheeOrg/Lychee/blob/b4cbfc4fe4aeab21b95ad14df666960ac2b2be9a/resources/views/includes/head.blade.php#L2-L7)). Hence, we started to load (and execute) or JS code before the CSS styles had been parsed. Now, the JS sources are included after the `<body>`.

But more importantly we now have the CSS classes `mode-none`, `mode-gallery`, `mode-view` and `mode-frame` for the `<body>` tag and the initial class for `<body>` is `mode-none`. This hides all potentially problematic parts such as the header. For example, we have the following CSS rule for header

```scss
body.mode-frame,
body.mode-none {
	.header {
		display: none;
	}
}
```

Unless the JS code in `lychee.load` has been executed and activated a particular mode, nothing is visible. Hence, ` $("html").css("visibility", "visible");` would be redundant. I would like to not add it, not only due to its redundancy, but also because I would like to avoid any direct CSS rule on an element.

Reply:
OK! We should remove that one line from `init.js` then, right?

Reply:
Sure. Fixed in https://github.com/LycheeOrg/Lychee-front/pull/323/commits/4a0180341721d03b6f6a0e82e601a04667080ca2.

Reply:
Unfortunately, I was wrong. We need it.

My error was to assume that the icons of the header are also added dynamically to the toolbar. Then my argument about the JS being loaded last would have hold. But the icons are statically coded into the main HTML markup. This may lead to a situation where the browser starts rendering the HTML page (incl. the icons), but the CSS is not yet loaded. Also the icons are part of the main file, but the CSS is included via `<link>` and hence the CSS is asynchronously loaded after the main file.

> The reason was that at least sometimes when loading Lychee, the page would start to render before CSS was loaded

One can (nearly deterministically) provoke that effect, if one artificially throttles the bandwidth and increases the delay via the developer tools of the browser. In Firefox, the preset "2G regular" did the trick.

I add a fix similar to the old one again in https://github.com/LycheeOrg/Lychee-front/pull/323/commits/3a17fe5829f2a32fae74aefa1abb84bb633afd01.

I wanted to avoid inline styles, but for the moment the old workaround is the best.